README.md

```diff
@@ -14,10 +18,57 @@ connections and responses from end endpoint.
 
 (TODO: some words to go along with these diagrams)
 
-#### Development setup
+### Development setup
+
+#### Node.js
+
+If you want to install Node.js natively visit [Node.js](http://nodejs.org), please make sure it is version 4.4.4
+
+To check the locally installed version run: 
+
+```bash
+node --version
 ```
-git clone https://github.com/department-of-veterans-affairs/healthcare-application.git
-cd healthcare-application
+
+Once you have version 4.4.4 of Node.js installed to required version of npm run:
+
+```bash
+npm i -g npm@3.8.9
+```
+
+You may want to use nvm (Node Version Manager), this will allow for managing and installing multiple versions of Node.js on your machine.
+
+You can install nvm from [here](https://github.com/creationix/nvm)
+
+If you are a [homebrew](http://brew.sh/) homebrew user you can install nvm by typing:
+
+```bash
+brew update && brew install nvm
+```
+
+Once you have nvm to installed to install version 4.4.4 of Node.js run: 
+
+```bash
+nvm install 4.4.4
+```
+
+Once you have version 4.4.4 of Node.js installed to required version of npm run:
```

Comment:
I think you can delete the previous instructions for how to require the version of npm with the new workflow and have that come after Node is installed.